.github/workflows/action-ci.yml

```diff
@@ -65,64 +65,29 @@ jobs:
           BRANCH=$(git rev-parse --abbrev-ref HEAD)
           echo "BRANCH is $BRANCH"
           echo "branch=$BRANCH" >> $GITHUB_OUTPUT
-
-  gaugerelease:
-    name: Check unreleased changes
+  
+  dispatchrelease:
+    name: Dispatch tag release
```

Comment:
But it's not actually doing the release, it's dispatching a workflow.

Reply:
I guess.  Perhaps then change it to `Dispatch tag patch release` so that it matches the dispatched workflow name then. I've recommended below to change the name of the step of the job to also match the  dispatched workflow, it's OK the job + step have the same name

Reply:
Done